core/src/main/scala/quasar/blobstore/services.scala

```diff
@@ -25,14 +25,16 @@ import fs2.Stream
 
 object services {
 
-  type StatusService[F[_]] = F[BlobstoreStatus]
+  type DeleteService[F[_]] = Kleisli[F, BlobPath, Int]
```

Comment:
Maybe improve this, so that it's clearer what the `Int` here represents and make it so that other blobstores can also implement the same interface? I thought about a newtype wrapper, but that's not going to work for other blobstores.

So maybe map the different values to an enum like 
```
DeleteStatus = Success | NotFound | NoAccess | ...
```
a la `BlobstoreStatus`?

Reply:
Sounds good!